packages/dev/cypress/e2e/unovis.cy.ts

```diff
@@ -0,0 +1,358 @@
+describe('Unovis Test', () => {
+  beforeEach(() => {
+    cy.visit('/')
+  })
+
+  it('Load homepage', () => {
+    cy.percySnapshot('Homepage test')
+  })
+
+  it('Basic Annotation', () => {
+    cy.contains('Basic Annotations').click()
+    cy.wait(3000)
```

Comment:
Is there a specific reason to wait 3000 ms?

Reply:
> Is there a specific reason to wait 3000 ms?

Some page has animations. This way we wait till everything is fully rendered before taking the screenshot.

Reply:
Should we add `duration` as a query string argument to the pages? I think this way you can speed up the testing significantly.

Reply:
> Should we add `duration` as a query string argument to the pages? I think this way you can speed up the testing significantly.

What do you mean? Add something like `?duration=3000` to the url?

Reply:
More like `?duration=0` to render charts without animation

Reply:
> More like `?duration=0` to render charts without animation

That's a good idea.

Reply:
> More like `?duration=0` to render charts without animation

See a POC [here](https://github.com/f5/unovis/pull/419/files#r1706006409)